core/src/main/resources/lib/form/checkbox.jelly

```diff
@@ -68,7 +68,9 @@ THE SOFTWARE.
          title="${attrs.tooltip}"
          onclick="${attrs.readonly=='true' ? 'return false;' : attrs.onclick}" id="${attrs.id}" class="${attrs.class} ${attrs.negative!=null ? 'negative' : null} ${attrs.checkUrl!=null?'validated':''}"
          checkUrl="${attrs.checkUrl}" checkDependsOn="${attrs.checkDependsOn}" json="${attrs.json}"
-         checked="${value ? 'true' : null}"/>
+         checked="${value ? 'true' : null}"
+         disabled="${readOnlyMode ? 'true' : null}"
+  />
```

Comment:
This diff can easily be minimized by just putting `disabled` on the line above `checked`. No need to put `/>` on a new line.